tests/build_plugin.rs

```diff
@@ -0,0 +1,24 @@
+use std::path::Path;
+
+#[test]
+fn test_example_builds() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
Yeah, I believe that because the `build_plugin` function will return an error when the build fails, this should be okay. A failed build will fail the test even though the file maybe still exists from prior builds. And then a successful build that doesn't put the output in the right place will also fail the build, particularly for a CI/CD environment.